metricbeat/module/kubernetes/_meta/test/README.md

```diff
@@ -15,4 +13,12 @@ suffix: plain
 path: "../_meta/test"
 ```
 
-Since all metricsets use a `data.json` file for the documentation - this file contains an example of the metrics fields -, **it is necessary that one of these files has the name `docs.plain`**. This file should have the **same content as one of the metrics file of a KSM version** we support. This means that one of the files is duplicated, for example: `ksm.v2.7.0.plain` has the same content as `docs.plain`. This is not a mistake, as having the `ksm.v2.7.0.plain` tells us the metrics of that specific version.
+
+When you update the KSM directory files, remember to run `go test -data` to generate the expected files. To check against the expected files already present, you can just run `go test .` inside each state metricset directory.
+
+> **TIP**: To run tests and generate the expected files for all state metricsets you can run `go test ./state... --data`.
+
+> **_NOTE:_**  The expected files inside these two folders are not deleted when running the tests. Remember to delete them if they are from an old version.
```

Comment:
```suggestion
> **_NOTE:_**  The expected files inside the two folders of each state_* mericset (`_meta/test` and `_meta/testdata`) are not deleted when running the tests. Remember to delete them if they are from an old version.
```

Reply:
@constanca-m testdata folder is not created from ksm tests

It seems that integration tests only needed so trying: `MODULE="kubernetes" mage goIntegTest` but the command also fails for me.

So maybe consider removing the reference `these two folders` overall?

Reply:
> @constanca-m testdata folder is not created from ksm tests

Yes, it should already be present.